packages/hardhat-verify/README.md

```diff
@@ -230,6 +230,16 @@ await hre.run("verify:verify", {
 
 If the verification is not successful, an error will be thrown.
 
+Note that the `"verify:verify"` subtask will *always* run the etherscan verifier, even when you have the following in your hardhat config file:
```

Comment:
As the last published version of `hardhat-verify` this would be incorrect, see [here](https://github.com/NomicFoundation/hardhat/blob/%40nomicfoundation/hardhat-verify%402.0.4/packages/hardhat-verify/src/index.ts#L264). You can verify both providers with the same task, and control which one is executed through the config.

Reply:
Ah ok, if so this PR is redundant. Closing!